### ISSUE TITLE
Status of this project?

### ISSUE THREAD

**Issue by ghost** [btb/d2x#1]
I thought d2x was dead, and dxx is its successor, but I see lots of activity here.
The readme is something ancient, even mentions mac os 9!
You don't have a public email address either, so I'm asking here.
Do you intend to go with this anywhere?

**Comment by btb** [comment]
Yeah, it's still dead in comparison to -rebirth, I only started making
changes again recently, and I'm doing it purely for fun and practice.
Updating the readme would be a good idea, haha.

**Comment by btb** [comment]
I can still be reached via btb@icculus.org fyi

**Comment by kreatordxx** [comment]
I made releases for Mac OS 9 until I decided it was just too impractical. The Mac I would need to do it is sitting in my storage unit. Now that Rebirth requires C++11, it would be incredibly difficult to release for Mac OS 9 (and C++11 calls for at least Mac OS X 10.7 Lion).
